index.sh

```diff
@@ -20,29 +19,77 @@ case $(uname -m) in
   *) echo "Architecture $ARCH not supported by the installation script"; exit 1;;
 esac
 
+BIN="$HOME/bin"
+
 ##/=====================================\
 ##|     CLEAN OLD INSTALLATION |
 ##\=====================================/
-nigiri stop --delete &> /dev/null
-rm -f $BIN/nigiri
-rm -rf ~/.nigiri 
+
+if [ "$(command -v nigiri)" != "" ]; then
+  echo "Nigiri is already installed and will be deleted."
+  echo "Stopping Nigiri..."
+  if [ -z "$(nigiri stop --delete &>/dev/null)" ]; then
+    :
+  fi
+  echo "Removing Nigiri..."
+  rm -f $BIN/nigiri
+  rm -rf ~/.nigiri
```

Comment:
we need to remove also from /usr/local/bin and usr/bin old installation path if we want to migrate to $HOME/bin

Reply:
I think this is something that can be delegated to our beloved early users since this is something we should not concern from now on